src/lib/Util.ts

```diff
@@ -12,9 +12,11 @@ type CmdResult = {
 };
 
 export class Util {
-    public path: string = '';
+    public path: string = "";
+    public flowPath: string = "";
     constructor(public workspaceRoot: string, private logger: Logger) {
-        this.path = vscode.workspace.getConfiguration('git').get('path') || "";
+        this.path = vscode.workspace.getConfiguration("git").get("path") || "";
+        this.flowPath = vscode.workspace.getConfiguration("gitflow").get("path") || `"${this.path}" flow`;
         if (this.path.trim().length === 0) {
```

Comment:
Now this like might fail. Because if `"${this.path}" flow` will be used and `"${this.path}"` wil be nothing then it will be ` flow` and this line will think there is something but there isnot.